src/test/java/kanbanflow/steps/KanbanStep.java

```diff
@@ -155,10 +155,32 @@ public void theUserCloseModalWindows() {
 
     /**
      * Method to let description in the board.
+     *
      * @param Description for the board.
      */
     @Then("show the task description (.*) in the board")
-    public void showTheTaskDescriptionTaskInTheBoard(String Description) {
+    public void showDescriptionBoard(String Description) {
         context.getBoardEntities().getBoard().isTaskInBoard(Description);
+
```

Comment:
please delete the unnecessary blank space

Reply:
Done.